### ISSUE TITLE
[Accordion] labeled slider inside closed accordion make accordion init routine stuck

### ISSUE THREAD

**Issue by CmdrMahesh** [fomantic/Fomantic-UI#1140]
# I am facing an issue when I try to initialize slider inside accordion to make filters based on sliders.

## Fomantic-UI initialization failure inside hidden containers

The Fomantic-UI slider is working great when it is been initialized in the visible container but when I try to initialize them insider containers like accordions my browser's loaders seem to keep loading, but in case of visible containers or can say active accordion the slider gets initialized and the page loads completely in a second.

**Comment by CmdrMahesh** [comment]
Here is code that i'm using:

```
<div class="ui styled accordion">
  <div class="title">    
    <div><strong>Title 1</strong></div>
  </div>
  <div class="content">	    
    <div class="ui slider"></div>
  </div>
</div>
```
And to initialize i'm using this:
```
<script>
$('.ui.slider')  .slider();
</script>
```

**Comment by lubber-de** [comment]
You  have to initialize the Accordion itself, otherwise the slider (or any other content) is hidden
See: https://jsfiddle.net/gcmdhynL/
```javascript
$('.ui.accordion').accordion();
$('.ui.slider').slider({min: 0,max: 2,start:0,end:2,step:1});
```
![slider_in_accordion](https://user-images.githubusercontent.com/18379884/68043891-a0a6d600-fcd6-11e9-9977-14db1050b930.gif)

**Comment by CmdrMahesh** [comment]
sorry my bad, that I haven't fully shown my code but as now I have updated the fiddle [code](https://jsfiddle.net/mzoyb5ja/26/) you can see when labeled class is used with the slider it's causing the issue.


HTML:

```
  <div class="ui styled accordion">
      <div class="title">    
        <div><strong>Title 1</strong></div>
      </div>
      <div class="content">	    
        <div class="ui labeled ticked range slider"></div>
      </div>
      <div class="title">    
        <div><strong>Title 1</strong></div>
      </div>
      <div class="content">	    
        <div class="ui labeled ticked range slider"></div>
      </div>
  </div>
```

JS:

```
$('.ui.accordion').accordion();
$('.ui.slider').slider({min: 0,max: 10,start: 0,end: 10,step: 1}); 
```

**Comment by lubber-de** [comment]
😨 OK, confirmed.... the `labeled` class dirsturbs the accordion init routine ...

**Comment by CmdrMahesh** [comment]
thank you @lubber-de, sorry to make this hurry but may I know a piece of code that can fix this so I can continue with my project and apply the new dist when ready :)

**Comment by lubber-de** [comment]
Fixed by #1151

**Comment by CmdrMahesh** [comment]
Thanks @lubber-de, hope **2.8.0** get released soon with all great fixes and more features.
